src/app/task-list/task-list-item/task-list-item.component.ts

```diff
@@ -9,24 +10,37 @@ import { TaskItem } from '../../models/TaskItem';
 export class TaskListItemComponent {
 
   public isTaskEditing: boolean;
+  modalRef: BsModalRef;
 
   @Input() task: TaskItem;
-  @Output() onTaskEdit = new EventEmitter<TaskItem>();
-  @Output() onTaskDelete = new EventEmitter<number>();
+  @Output() taskEdit = new EventEmitter<TaskItem>();
+  @Output() taskDelete = new EventEmitter<number>();
+  @Output() taskOpenModel = new EventEmitter<any>();
+  constructor( private modalService: BsModalService){
```

Comment:
Add space between constructor and last output

Reply:
done